lib/Selenium/Remote/Driver.pm

```diff
@@ -144,6 +144,7 @@ you please.
         'javascript'           - <boolean>  - whether javascript should be supported
         'accept_ssl_certs'     - <boolean>  - whether SSL certs should be accepted, default is true.
         'firefox_profile'      - Profile    - Use Selenium::Firefox::Profile to create a Firefox profile for the browser to use
+        'marionette_enabled'   - <boolean>  - whether Firefox should enable Marionette. default if false
```

Comment:
These options are for the Selenium::Remote::Driver which doesn't do the binary start up, so I don't think this option belongs here. Unless you mean that passing `marionette_enabled` to a running `selenium-standalone-server-x.xx.x.jar` would start marionette mode, in which case, never mind :D